src/store/mutations.ts

```diff
@@ -329,9 +330,18 @@ class Mutations {
    */
   setVariableDelimiters(
     state: VQBState,
-    { variableDelimiters }: Pick<VQBState, 'variableDelimiters'>,
+    {
+      variableDelimiters,
+      advancedVariableDelimiters,
+    }: {
+      variableDelimiters: VariableDelimiters;
+      advancedVariableDelimiters: VariableDelimiters;
+    },
   ) {
     state.variableDelimiters = variableDelimiters;
+    if (advancedVariableDelimiters) {
```

Comment:
Why testing this ? If `state.advancedVariableDelimiters` is undefined, what's the deal ?

Reply:
Or two different mutations.

Reply:
Because we can have variableDelimiters but no advancedVariableDelimiters maybe, if an app doesn't want to use advanced variable he won't provide the second param. We can make two different mutations if you prefer, but i think that in a few days/weeks we will use all the variable delimiters as one and only object (as i said in the description) that's why it thought adding a new mutation would be a waste of time!